blueprint/src/content.tex

```diff
@@ -505,7 +505,9 @@ \section{Simplicial sets}\label{sec:simplicial-sets}
 
 \begin{definition}[isofibration]\label{defn:qcat-isofibration}
   \uses{defn:coherent-isomorphism}
-  A simplicial map $f \colon A \to B$ between quasi-categories is an \textbf{isofibration} if it lifts against the inner horn inclusions, as displayed below-left, and also against the inclusion of either vertex into the free-living isomorphism $\iso$.
```

Comment:
Let's either revert this change or add a footnote to clarify. You and I know it is more efficient but I worry it won't be clear to everyone that $A$ is necessarily also a quasi-category. You can pick which.